var/spack/repos/builtin/packages/llvm-openmp/package.py

```diff
@@ -76,10 +93,31 @@ def patch(self):
             os.rename(cmake_mod_dir, os.path.join(self.stage.path, "cmake"))
 
     def cmake_args(self):
-        cmake_args = []
+
+        # Disable LIBOMP_INSTALL_ALIASES, otherwise the library is installed as
+        # libgomp alias which can conflict with GCC's libgomp.
+        cmake_args = ["-DLIBOMP_INSTALL_ALIASES=OFF"]
+        cmake_args.append("-DOPENMP_STANDALONE_BUILD=1")
         # Add optional support for both Intel and gcc compilers
         if self.spec.satisfies("+multicompat"):
             cmake_args.append("-DKMP_GOMP_COMPAT=1")
```

Comment:
```suggestion
        if self.spec.satisfies("+multicompat"):
            cmake_args.append("-DKMP_GOMP_COMPAT=1")            
            # Disable LIBOMP_INSTALL_ALIASES, otherwise the library is installed
            # as libgomp alias which can conflict with GCC's libgomp.
            cmake_args.append("-DLIBOMP_INSTALL_ALIASES=OFF")

```

Reply:
I made this change.